packages/core/src/portable.ts

```diff
@@ -13,9 +14,22 @@ export interface PortableResult {
 
 export interface PortableHint {
   requireFile: string;
+  packageVersion: string | undefined;
   useMethod?: string;
 }
 
+const { findUpPackagePath } = resolvePackagePath;
+
+export function maybeNodeModuleVersion(path: string) {
+  const packagePath = findUpPackagePath(path);
+
+  if (packagePath === null) {
+    throw new Error(`Could not find package.json for '${path}'`);
```

Comment:
```suggestion
    throw new Error(`[Embroider] Could not find package.json for '${path}'`);
```

Just for consistency with the same prefix as you had for the relative path babel plugin...